### DISCUSSION TITLE
topic怎么路由到新增的broker节点？

### DISCUSSION THREAD

**Discussion by liyale123** [apache/rocketmq#7046] [General]
rocketmq新增了一个broker节点，之前创建的topic怎么能同步路由到新节点呢？有没有相关命令？dashboard需要手动选，topic太多了。。。

**Comment by RongtongJin** [comment]
> rocketmq新增了一个broker节点，之前创建的topic怎么能同步路由到新节点呢？有没有相关命令？dashboard需要手动选，topic太多了。。。

Copy the topics.json and subscriptionGroup.json from other nodes to the newly added broker node, and then restart.
